packages/gafl-webapp-service/src/locales/en.json

```diff
@@ -79,7 +79,7 @@
   "address_lookup_content_info": "This is where we will send the licence card. A bankside enforcement officer may ask for this information if the licence is checked.",
   "address_lookup_crown_copyright_new_tab": "OS (opens in a new tab)",
   "address_lookup_crown_copyright_ref": "100024198.",
-  "address_lookup_crown_copyright_terms_link": "terms and conditions",
```

Comment:
I think we'll need this in Welsh too. I've asked Joe and Melinda to confirm what they want, but I think the Welsh translation of "telerau a'r amodau" should be amended to "telerau a'r amodau (yn agor ar dudalen newydd)"

Reply:
@jaucourt Since we've agreed to handle the translation in a separate ticket, is it OK to merge this PR now?

Reply:
Yes - approved